### ISSUE TITLE
The 3.0.10 release jar is mostly empty

### ISSUE THREAD

**Issue by henrik242** [cbeust/klaxon#202]
```
$ wget http://jcenter.bintray.com/com/beust/klaxon/3.0.10/klaxon-3.0.10.jar
$ unzip -l klaxon-3.0.10.jar 
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  10-02-2018 13:17   META-INF/
        0  10-02-2018 13:17   META-INF/MANIFEST.MF
        0  10-02-2018 13:17   klaxon/
        0  10-02-2018 13:17   klaxon/kobaltBuild/
        0  10-02-2018 13:17   klaxon/kobaltBuild/classes/
        0  10-02-2018 13:17   klaxon/kobaltBuild/classes/com/
        0  10-02-2018 13:17   klaxon/kobaltBuild/classes/com/beust/
        0  10-02-2018 13:17   klaxon/kobaltBuild/classes/com/beust/klaxon/
        0  10-02-2018 13:17   com/
        0  10-02-2018 13:17   com/beust/
        0  10-02-2018 13:17   com/beust/klaxon/
      209  10-02-2018 13:17   com/beust/klaxon/KlaxonDoc.class
---------                     -------
      209                     12 files
```

**Comment by cbeust** [comment]
Yes, will fix this as soon as I get a chance.

**Comment by cbeust** [comment]
Fixed in 3.0.11.
